lib/git-fastclone.rb

```diff
@@ -144,12 +144,16 @@ def clone(url, rev, src_dir)
       initial_time = Time.now
 
       with_git_mirror(url) do |mirror|
-        Cocaine::CommandLine.new("git clone --quiet --reference '#{mirror}' '#{url}'" \
-          " '#{File.join(abs_clone_path, src_dir)}'").run
+        Cocaine::CommandLine.new('git clone --quiet --reference', ':var')
```

Comment:
I think it would be easier to read with separate named args, i.e.

    Cocaine::CommandLine.new('git clone', '--quiet --reference :mirror :repository :directory')
      .run(mirror: mirror, repository: url, directory: File.join(abs_clone_path, src_dir))